src/basilisp/contrib/sphinx/autodoc.py

```diff
@@ -147,7 +147,8 @@ def filter_members(
         self, members: List[ObjectMember], want_all: bool
     ) -> List[Tuple[str, Any, bool]]:
         filtered = []
-        for name, val in members:
+        for member in members:
+            name, val = member.__name__, member.object
```

Comment:
Was getting this deprecation warning:
```
/home/chris/Projects/basilisp/src/basilisp/contrib/sphinx/autodoc.py:387: RemovedInSphinx80Warning: The tuple interface of ObjectMember is deprecated. Use (obj.__name__, obj.object) instead.
  for name, val in members:
```

Fortunately, the earliest version of Sphinx currently supported is `^7.1.0` and these attributes existed [as of that release](https://github.com/sphinx-doc/sphinx/blob/e560f6334aa87af07304415e6aeaf42d03d1c525/sphinx/ext/autodoc/__init__.py#L280-L286).